aws-fluentd-cell.conf

```diff
@@ -13,10 +13,6 @@
 </source>
 
 <label @containers>
-  <filter **>
-    @type kubernetes_metadata
-    @id filter_kube_metadata
-  </filter>
 
   <filter **>
     @type record_transformer
```

Comment:
I think `kubernetes_metadata` is also used for another label later in this config file.